### PR TITLE
Extensions of SAML Auth Context Ext

### PR DESCRIPTION
**Pull request by Razumain**
Updates to allow inclusion of extended authentication information in AuthContextExtensions

### CODE REVIEW

File: ELN-0603 - Registry for Identifiers.md [hunk 1]
```diff
@@ -330,6 +330,7 @@ Identifiers associated with the Authentication Context X.509 extension
 | **URL** | **Object** | **Reference** |
 | :--- | :--- | :--- |
 | `http://id.elegnamnden.se/auth-cont/1.0/saci` | XML schema name space for SAML Authentication Context Information in the Authentication Context X.509 certificate extension | \[[AuthContExt](#authcontext)\] |
+| `http://id.swedenconnect.se/auth-cont/1.0/ext-auth-info` | XML schema name space for Extended Authentication Information in the Authentication Context X.509 certificate extension | \[[CertProf](#certprof)\] |
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: It's namespace, not name space ...
Fix in both places.

File: ELN-0603 - Registry for Identifiers.md [hunk 2]
```diff
@@ -546,6 +547,8 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
 
 - The Sign Message Authentication Context URIs defined in section 3.1.1.1 are deprecated.
 
+- Added XML Schema name space identifier `http://id.swedenconnect.se/auth-cont/1.0/ext-auth-info`. This XML Schema defines XML extension to the Authentication Context Extension to provide extended authentication information.
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: "name space" -> namespace

This XML Schema defines **an** XML extension **for** (not to) the ...

File: ELN-0608 - Certificate Profile for Central Signing Services.md [hunk 3]
```diff
@@ -183,6 +183,52 @@ The `<saci:IdAttributes>` element SHALL contain one `<saci:AttributeMapping>` el
 other name form that was obtained from a SAML attribute in the SAML assertion used to authenticate the signer as part of the signature creation process. Each `<saci:AttributeMapping>` element SHALL provide the `<saml:AttributeValue>` that
 were obtained from the SAML assertion.
 
+##### 2.3.2.1 Extended Authentication Information
+
+In addition to the attributes of `<sci:AuthContextInfo>` it is possible to provide additional authentication information through the extensibility of `<sci:AuthContextInfo>` which allows inclusion of a sequence of any element.
+
+One such element is defined in this section, the `<sacex:ExtAuthInfo>` element.
+
+This element MAY be included to provide a name of a parameter and its associated value. This element MAY be used to carry the value of any single valued attribute from the associated SAML Assertion as long as the SAML attribute value is not composed of a complex type. When used to carry a SAML attribute value, the value of the `<sacex:ExtAuthInfo>` element SHALL be identical to the content of the SAML attribute value element and the Name attribue SHALL hold the same value as the Name attribute of the corresponding SAML attribute.
```
Comment (martin-lindstrom) [inline ELN-0608 - Certificate Profile for Central Signing Services.md]: When referring to the Name attribute, make it in code-format, `Name`.

File: ELN-0603 - Registry for Identifiers.md (hunk 1 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: OK

File: ELN-0603 - Registry for Identifiers.md (hunk 2 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: OK

File: ELN-0608 - Certificate Profile for Central Signing Services.md (hunk 3 above)
Comment (Razumain) [reply ELN-0608 - Certificate Profile for Central Signing Services.md]: Fixed

**Pull request merged**